frappe/email/__init__.py

```diff
@@ -20,10 +20,10 @@ def get_contact_list(txt, page_length=20) -> list[dict]:
 	match_conditions = f"and {reportview_conditions}" if reportview_conditions else ""
 
 	out = frappe.db.sql(
-		f"""select email_id as value,
+		f"""select name as value, email_id as label,
```

Comment:
This doesn't seem right? The value should be email? `name` isn't a valid email


 https://github.com/frappe/frappe/blob/56fb0a4f93be977c3bdd70fd886e5f6efb45276c/frappe/contacts/doctype/contact/contact.py#L14-L22


am I missing something?

Reply:
Apparantly this is how it works, I still don't get how :woozy_face: 

value should be actual value and label user facing... usually.